src/cli_common.c

```diff
@@ -318,36 +326,26 @@ void parseRedisUri(const char *uri, const char* tool_name, cliConnInfo *connInfo
     UNUSED(tls_flag);
 #endif
 
-    const char *scheme = "valkey://";
-    const char *tlsscheme = "valkeys://";
-    /* We need to support redis:// and rediss:// too for compatibility. */
-    const char *redisScheme = "redis://";
-    const char *redisTlsscheme = "rediss://";
```

Comment:
Please don't do unnecessary refactoring. These are fine as const char.

> it would be better if we avoid redis even in variable names to stay truly unique going forward

I don't agree. For the scheme "redis://" using a variable name like PRE_VALKEY is just ridiculous. It's the redis scheme. Let's call it what it is.

Reply:
Okay reverted those changes, please see the latest commit.